tests/plugins/emoji/manual/emoji.html

```diff
@@ -0,0 +1,16 @@
+<textarea id="editor1" cols="10" rows="10">
+	<p>Hello world.</p>
+</textarea>
+
+<p>Empty editor</p>
```

Comment:
Why is it called "Empty editor" when it's not empty? 🙂 It's a divarea editor. First one should be named too for consistency.

Also use headers for that.

Reply:
This issue might occur in other places too.